packages/plugin-core/src/test/suite-integ/ShowLegacyPreview.test.ts

```diff
@@ -0,0 +1,93 @@
+import { DendronWebViewKey } from "@dendronhq/common-all";
+import { ENGINE_HOOKS } from "@dendronhq/engine-test-utils";
+import { test } from "mocha";
+import sinon from "sinon";
+import { ExtensionContext } from "vscode";
+import { ShowLegacyPreviewCommand } from "../../commands/ShowLegacyPreview";
+import { VSCodeUtils } from "../../utils";
+import { MarkdownUtils } from "../../utils/md";
+import { getExtension } from "../../workspace";
+import { expect } from "../testUtilsv2";
+import {
+  runLegacyMultiWorkspaceTest,
+  setupBeforeAfter,
+  withConfig,
+} from "../testUtilsV3";
+
+suite("ShowLegacyPreview", function () {
+  let ctx: ExtensionContext;
+  ctx = setupBeforeAfter(this);
+
+  test("ok: show previewv2 by default", (done) => {
+    runLegacyMultiWorkspaceTest({
+      ctx,
+      preSetupHook: ENGINE_HOOKS.setupBasic,
+      onInit: async ({ engine }) => {
+        const note = engine.notes["foo"];
+        await VSCodeUtils.openNote(note);
+        await new ShowLegacyPreviewCommand().execute();
+        const preview = getExtension().getWebView(
+          DendronWebViewKey.NOTE_PREVIEW
+        );
+        expect(preview?.visible).toBeTruthy();
+        done();
+      },
+    });
+  });
+
+  test("ok: prompt install for legacy preview when missing", (done) => {
+    runLegacyMultiWorkspaceTest({
+      ctx,
+      preSetupHook: ENGINE_HOOKS.setupBasic,
+      onInit: async ({ engine, wsRoot }) => {
+        withConfig(
+          (config) => {
+            config.dev = {
+              enablePreviewV2: false,
```

Comment:
this configuration should no longer be checked for

Reply:
`promptInstallLegacyPreview` will be unreachable anyways since it won't ever be called when MPE isn't installed 😄  I'll remove this test